src/components/Facility/AssetCreate.tsx

```diff
@@ -55,17 +57,35 @@ const formErrorKeys = [
   "support_email",
   "manufacturer",
   "warranty_amc_end_of_validity",
-  "last_serviced_on",
+  "last_serviced",
   "notes",
 ];
 
-const initError = formErrorKeys.reduce(
-  (acc: { [key: string]: string }, key) => {
-    acc[key] = "";
-    return acc;
-  },
-  {},
-);
+interface AssetData {
+  id?: string;
+  name?: string;
+  location?: string;
+  description?: string;
+  is_working?: boolean | null;
+  not_working_reason?: string;
+  created_date?: string;
+  modified_date?: string;
+  serial_number?: string;
+  asset_type?: AssetType;
+  asset_class?: AssetClass;
+  status?: "ACTIVE" | "TRANSFER_IN_PROGRESS";
+  vendor_name?: string;
+  support_name?: string;
+  support_email?: string;
+  support_phone?: string;
+  qr_code_id?: string;
+  manufacturer?: string;
+  warranty_amc_end_of_validity?: string;
+  serviced_on?: string;
+  latest_status?: string;
+  last_service?: any;
+  notes: string;
+}
```

Comment:
This type is already defined in our code base. Let's not redefine it. Import the existing type instead.

Reply:
@rithviknishad existing AssetData type have serviced_on date and note encapsulated in last_service object, making it incompatible to be used with Form provider

Reply:
@rithviknishad that type contains last_service object which has notes and on_service date , but we have individual fields for those in form that uses Provider